web/src/components/molecules/Asset/Asset/AssetBody/svgPreview.tsx

```diff
@@ -7,7 +7,12 @@ const SVGPreview: React.FC<Props> = ({ url, svgRender }) => {
   const [svgText, setSvgText] = useState("");
 
   const fetchData = useCallback(() => {
-    fetch(url)
+    fetch(url, {
+      method: "get",
```

Comment:
method `get` should be upper-case by convention. (to be `GET`)

Reply:
Done!